pandas/tests/tseries/frequencies/test_freq_code.py

```diff
@@ -30,18 +32,18 @@ def test_get_to_timestamp_base(freqstr, exp_freqstr):
         ("M", "month"),
         ("D", "day"),
         ("H", "hour"),
-        ("T", "minute"),
+        ("min", "minute"),
         ("S", "second"),
-        ("L", "millisecond"),
-        ("U", "microsecond"),
-        ("N", "nanosecond"),
+        ("ms", "millisecond"),
+        ("us", "microsecond"),
+        ("ns", "nanosecond"),
     ],
 )
 def test_get_attrname_from_abbrev(freqstr, expected):
     assert Resolution.get_reso_from_freqstr(freqstr).attrname == expected
 
 
-@pytest.mark.parametrize("freq", ["D", "H", "T", "S", "L", "U", "N"])
+@pytest.mark.parametrize("freq", ["D", "H", "min", "S", "ms", "us", "ns"])
```

Comment:
is it weird to have "S" instead of "s"?  probably not a problem

Reply:
yes, I agree. I'll add to this PR the replacing “S” with “s”